frontend/server/src/Controllers/Course.php

```diff
@@ -5869,7 +5875,15 @@ public static function apiRuns(\OmegaUp\Request $r): array {
             $r->ensureOptionalEnum('language', $languages),
             !is_null($identity) ? $identity->identity_id : null,
             max($r->ensureOptionalInt('offset') ?? 0, 0),
-            $r->ensureOptionalInt('rowcount') ?? 100
+            $r->ensureOptionalInt('rowcount') ?? 100,
+            $r->ensureOptionalEnum(
+                'execution',
+                \OmegaUp\Controllers\Run::EXECUTION
```

Comment:
Si esto lo conviertes a array asociativo, ahora sólo tendrías que obtener las llaves del array:
```suggestion
                array_keys(\OmegaUp\Controllers\Run::EXECUTION)
```